joining.html

```diff
@@ -0,0 +1,217 @@
+      Your recruiter will ask you to sign up for
+      <a href="https://esiknife.com/">ESI Knife</a>, which is a website that
+      allows your recruiter to look at your character's history, assets, and
+      so on. This is a standard security precaution for Eve corps and does
+      <b><u>not</u></b> grant any access to do anything other than view
+      details.
+    </p>
+
+    <h2>
+      <a name="interview" href="#interview">#</a>
+      Interview
+    </h2>
+    <p>
+      After your trial fleet, your recruiter will work with you to schedule
+      an interview over Teamspeak. This interview generally lasts half an
+      hour at most. You'll probably be asked about at least these things:
+      <ul>
+        <li>Your experiences in Eve</li>
+        <li>Your corp history</li>
+        <li>Your trial fleet experience</li>
+      </ul>
+    </p>
+
+    <h2>
+      <a name="invitation" href="#invitation">#</a>
+      Invitation
+    </h2>
+    <p>
+      Once your trial fleet, ESI Knife audit, and interview are complete, your
+      recruiter will discuss your application with the other recruitment
+      staff. If everything looks good, your recruiter will invite you to join
+      the corp.
+    </p>
+
+    <h2>
+      <a name="joining" href="#joining">#</a>
+      Joining In Game
+    </h2>
+    <p>
+      If you're invited to join, you can now apply to join <b>Nobody in
+      Local</b> in game. A director will give your application a final check,
+      then accept your in-game application. Feel free to wait to confirm joining
+      in game if you need time to move assets or similar.
+    </p>
+
+    <h2>
+      <a name="onboarding" href="#onboarding">#</a>
+      Onboarding
+    </h2>
+    <p>
+      Once you've joined in game, head on over to <a href="onboarding.html">
+      onboarding</a> to get set up!
+    </p>
+
+    <h2>
+      <a name="questions" href="#questions">#</a>
+      Questions You Might Have
+    </h2>
+    <h3>
+      <a name="why-sp-max" href="#why-sp-max">#</a>
+      Why is there an SP maximum?
+    </h3>
+    <p>
+      Short answer: because we want SOUND to be a group focused on teaching
+      and learning.
+    </p>
+    <p>
+      Long answer: while we know that players of every SP level have things to
+      learn in Eve, our past experience has been that taking a lot of high-SP
+      players makes it more difficult for low-SP players to ask questions, and
+      can lead to a "bittervet" atmosphere of negativity about the game. That
+      isn't fun for anyone.
+    </p>
+    <p>
+      If you're over the 15M SP maximum, don't worry - you can still join, but
+      there's an extra step. Hop into our public channel or join a couple of
+      our regular public fleets, and try to get to know a couple of existing
+      members of the corp. We're looking for culture fit here a lot more than
+      pilot skill - it's fine if you're bad at the game, as long as you have a
+      good attitude about it.
```

Comment:
We'll need to schedule fleets more regularly or somehow communicate this information to interested parties if they would need to join multiple fleets.

Reply:
Yep, it's true. I have an idea for how to do that but it is not fleshed out yet - I need to talk to the recruiters some about it.